### PR TITLE
Origin/feature/MC-17820 Create Subnetworks Docs

### PR DESCRIPTION
**Pull request by emilyroser**
#### Changes made

- Added docs to create a list of subnetworks for a provided VPC

#### Related PRs
- [PR#209](https://github.com/cloudops/cloudmc-aws-plugin/pull/209)

### CODE REVIEW

File: source/includes/aws/_subnetworks.md [hunk 1]
```diff
@@ -123,6 +123,60 @@ Retrieve a subnetwork in a given [environment](#administration-environments).
...
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/subnetworks</code>
+
+Create a VPC in a given [environment](#administration-environments).
+
+| Attributes                      | &nbsp;                                              |
+| ------------------------------- | --------------------------------------------------- |
+| `vpcId`<br/>_string_            | The ID of the VPC.                                  |
+| `subnetsToCreate`<br/>_string_  | List of subnets to be created for a VPC             |
```
Comment (joelplourde4) [inline source/includes/aws/_subnetworks.md]: I am not 100% sold on the notation here, but it has nothing to do with this PR

**Review by joelplourde4 — APPROVED** [review]
"I'm afraid for the calendar. Its days are numbered."

File: source/includes/aws/_subnetworks.md [hunk 2]
```diff
@@ -123,6 +123,60 @@ Retrieve a subnetwork in a given [environment](#administration-environments).
...
 | `vpcId`<br/>_string_                               | The ID of the VPC the subnet is associated with.                                                                                |
 
+<!-------------------- CREATE SUBNETWORKS -------------------->
+
+#### Create Subnetworks
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/aws/test-env/subnetworks?operation=create"
```
Comment (dpmurray21) [inline source/includes/aws/_subnetworks.md]: You shouldn't need the query param here

File: source/includes/aws/_subnetworks.md [hunk 3]
```diff
@@ -123,6 +123,60 @@ Retrieve a subnetwork in a given [environment](#administration-environments).
...
+```json
+{
+  "taskId": "7135ae25-8488-4bc5-a289-285c84a00a84",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/subnetworks</code>
+
+Create a VPC in a given [environment](#administration-environments).
```
Comment (dpmurray21) [inline source/includes/aws/_subnetworks.md]:
```suggestion
Create a subnetwork in a given [environment](#administration-environments).
```

File: source/includes/aws/_subnetworks.md (hunk 1 above)
Comment (dpmurray21) [reply source/includes/aws/_subnetworks.md]: Type should be a list of objects here instead of a string

**Review by dpmurray21 — CHANGES_REQUESTED** [review]

**Review by dpmurray21 — APPROVED** [review]

**Pull request merged**